opencensus/trace/internal/trace_config.cc

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "opencensus/trace/internal/trace_config_impl.h"
 #include "opencensus/trace/trace_config.h"
+#include "opencensus/trace/internal/trace_config_impl.h"
```

Comment:
This seems to be a bug in the clang-format. Can you report?

Reply:
1. I think the problem here is we should have had a blank line after the _impl include.

2. Travis has clang-format v3.9.0 (Sep 2016), which is different from what I have on my workstation, and produces different results. My workaround was to make format.sh look for clang-format-3.9. Devs on Debian systems can simply "apt install clang-format-3.9"

Reply:
1. On second thoughts, isn't this the correct order? trace_config.cc includes trace_config.h first.

Reply:
Then shouldn't be an empty line between the main ".h" include and the rest of the includes?

Reply:
Done. :)